src/js/components/Anchor/Anchor.js

```diff
@@ -34,13 +35,14 @@ const Anchor = forwardRef(
       onClick: onClickProp,
       onFocus,
       reverse,
+      size: sizeProp,
       ...rest
     },
     ref,
   ) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
     const [focus, setFocus] = useState();
-
+    const size = useContext(TextSizeContext);
```

Comment:
Make context name a bit more generic/flexible? Not that we are locking ourselves into anything here since this implementation specific.

```suggestion
    const { size } = useContext(TextContext);
```